lmdeploy/cli/chat.py

```diff
@@ -31,6 +31,7 @@ def add_parser_torch():
         ArgumentHelper.session_len(engine_group)
         ArgumentHelper.adapters(engine_group)
         ArgumentHelper.cache_max_entry_count(engine_group)
+        ArgumentHelper.cache_block_seq_len(engine_group)
```

Comment:
In my opinion, it is not necessary to expose `cache_block_seq_len` in chat cli. 
`chat-cli` is used to check if the LLM inference works rather than demonstrate high performance. 
Users will barely care about `cache_block_seq_len` when they use this tool.

Reply:
Maybe it can be used to verify if the result is correct for a certain `cache_block_seq_len`.

Reply:
Make sense